pytmx/pytmx.py

```diff
@@ -651,7 +652,9 @@ def reload_images(self) -> None:
 
             path = os.path.join(os.path.dirname(self.filename), ts.source)
             colorkey = getattr(ts, "trans", None)
-            loader = self.image_loader(path, colorkey, tileset=ts)
+            kwargs_to_pass = self.image_loader_kwargs.copy()
+            kwargs_to_pass["tileset"] = "ts"
+            loader = self.image_loader(path, colorkey, **kwargs_to_pass)
```

Comment:
what is the point of `kwargs_to_pass`?

Reply:
As you see in line 704 and 716, I would be happy to pass the kwargs directly through.  Without passing the kwargs, the new alphaonly and the already-existing pixelalpha parameters do not get passed all the way from __init__() to the image loader and the values get lost.

The reason for the actual variable `kwargs_to_pass` here is to pass along the kwargs that were introduced in the __init__() but also inject the "ts" value into the dictionary to maintain its existing functionality.  I thought I could do it without using a variable but after much tinkering, I couldn't figure out a way.  Maybe my Python is weak.  :/  If there is a clean way to omit the variable itself (if that's what you're referring to), that was my original hope anyway.  If it's just an explanation you're after, I hope this makes it more clear.

Reply:
yeah, im still not understanding why there is a need for some additional dictionary.  could you illustrate the purpose in code?  or just drop it?

Reply:
I've removed the additional dictionary.